### PR TITLE
Add enum (unquoted values) support.

### PR DESCRIPTION
**Pull request by bol-com-pschmitz**
Add support for specifying enums as values resulting in unquoted enum names in the GraphQL.

### CODE REVIEW

File: core/build.gradle [hunk 1]
```diff
@@ -1,6 +1,6 @@
 buildscript {
     ext {
-        publish_version = '0.6.0'
+        publish_version = '0.7.0'
```
Comment (VerachadW) [inline core/build.gradle]: Could you revert this one?

**Comment by VerachadW** [comment]
I think all the changes is done in #25 . I'm not sure what is difference here.

**Comment by bol-com-pschmitz** [comment]
> I think all the changes are done in #25 . I'm not sure what is difference here.

I didn't see that, sorry. If that will cover the same functionality, feel free to close my pull request! Any idea when it will be released?

**Comment by VerachadW** [comment]
@bol-com-pschmitz Should be around next week. I will probably release it after Christmas.

**Pull request closed without merging**